boot-qemu.py

```diff
@@ -329,7 +329,8 @@ def get_linux_ver_code(decomp_cmd):
     for line in strings.stdout.decode("UTF-8").split("\n"):
         if re.search(r"Linux version \d+\.\d+\.\d+", line):
             # "Linux version x.y.z-<localversion> ..." -> "x.y.z-<localversion>" -> "x.y.z" -> ['x', 'y', 'z']
```

Comment:
Consider updating this comment too, to show what we can expect for input formats.

Reply:
Ack, will do so shortly